packages/eventual-send/NEWS.md

```diff
@@ -1,5 +1,13 @@
 User-visible changes in eventual-send:
 
+# v0.16.1 (2022-08-25)
+
+- Adds support `DEBUG=track-turns` and `TRACK_TURNS=enabled` environment
+  variables, such that turn tracking is disabled and tacit by default.
```

Comment:
"tacit"?

Reply:
I’ll change this to “quiet”. (I know the word in the context of music, where it borrows the meaning “silent” from Italian, preserving the original meaning from Latin. Your question prompted me to look it up, and evidently the English meaning evolved from a figurative sense of “implied”, which clearly is confusing here.)